### ISSUE TITLE
Not fully accessible with JAWS screen reader

### ISSUE THREAD

**Issue by howardgld** [trypsynth/taskmon#16]
The column headers and bottom status bar cannot be read with JAWS. There seems to be no way for a JAWS user to modify the column order. Suggest adding "move up" and "move down" controls to the column selection section of Settings.

**Comment by howardgld** [comment]
Sorry, my earlier comments were incorrect. I don't know what was wrong before but now I have no trouble seeing the column headers and status bar. I'd still like an easier way to reorder columns though.

**Comment by trypsynth** [comment]
Not sure on the JAWS thing, I'm an NVDA user myself, but taskmon is just standard Win32 controls, so I'd hope JAWS could handle it normally. I do agree about column ordering though and this is something I want to add.

**Comment by howardgld** [comment]
Okay. As I said, I don’t know why it wasn’t working a t first but it fine now with JAWS. I’m beginning to really love this program!

**Comment by trypsynth** [comment]
Column reordering is in as of bc98829.

Settings is now split into a General tab and a Columns tab. On the Columns tab you can reorder with Ctrl+Up and Ctrl+Down, or with the Move up and Move down buttons. The order is saved between runs. The Name column stays first.

Closing this since the JAWS side turned out to be a false alarm. Open a new issue if anything else comes up there.
